packages/rum-core/src/domain/resource/resourceUtils.spec.ts

```diff
@@ -313,3 +314,33 @@ describe('shouldTrackResource', () => {
     expect(isAllowedRequestUrl(configuration, 'https://my-domain.com/hello?a=b')).toBe(true)
   })
 })
+
+describe('findDataUrlAndTruncate', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think it could also be nice to test data url on other exotic [mime types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) like you do for svg images.  Ex : "application/vnd.openxmlformats-officedocument.presentationml.presentation"